dataactcore/models/baseInterface.py

```diff
@@ -161,8 +144,32 @@ def getNameFromDict(self, model, dictName, fieldName, fieldValue, idField):
         raise ValueError("Value: " + str(fieldValue) + " not found in dict: " + str(dict))
 
 
+def dbConnection():
+    """Use the config to set up a database engine and connection"""
+    if not CONFIG_DB:
+        raise ValueError("Database configuration is not defined")
+
+    dbName = CONFIG_DB['db_name']
```

Comment:
It will indeed, matching the current behavior. Do you think we should raise a `ValueError` instead?